docs/data-model.md

```diff
@@ -145,7 +145,7 @@ curl localhost:10101/index/repository/field/quantity \
 
 ##### BSI Range-Encoding
 
```

Comment:
Should we perhaps include a reference to the previous format, and a comment about the upgrade process? Seems like it'd be very nice for that to be present in case Something Goes Wrong.